### PR TITLE
Add onboarding steps for community managers

### PR DESCRIPTION
**Pull request by marcduiker**

### CODE REVIEW

**Review by msfussell — COMMENTED** [review]
Reviewed

File: community-manager-onboarding-2.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Dapr uses 1Password that contains credentials for the various accounts that are used by the Dapr team members.
+
+The community manager will receive an email from on of the project maintainers with an invite to join the Dapr 1Password account.
+
+### Dapr Community repo
+
+For planning and managing community calls, the community manager adds issues and edit them in the [community repo](https://github.com/dapr/community)
+
+Once new community calls have been planned, update the community repo [README.md](README.md) with the new dates.
```
[the remark this answers is not in the document]
Comment (marcduiker) [reply community-manager-onboarding-2.md:17]: I've updated the COMMUNITY-MANAGER.md to reflect this.

File: community-manager-onboarding-2.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+### Discord
```
[the remark this answers is not in the document]
Comment (marcduiker) [reply community-manager-onboarding-2.md:27]: I've updated the COMMUNITY-MANAGER.md with this info.

File: community-manager-onboarding-2.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+The Dapr Discord server is used for community discussions and Q&A. The responsibility of the community manager is to manage and moderate the Discord server.
+
+In case the community manager want to invite people to the Dapr Discord server, use this [invite link](https://bit.ly/dapr-discord).
+
+The _Community Sync_ DM group is used by community managers to discuss moderation and management tasks, and to plan community calls.
+
+### Twitter
+
+The [Dapr Twitter](https://twitter.com/daprdev) account is used for:
```
[the remark this answers is not in the document]
Comment (marcduiker) [reply community-manager-onboarding-2.md]: I've moved this to the COMMUNITY-MANAGER.md file.

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**